inc/Engine/Media/PreconnectExternalDomains/Frontend/Subscriber.php

```diff
@@ -42,4 +43,14 @@ public static function get_subscribed_events(): array {
 	public function preconnect_domains( array $items ) {
 		return $this->controller->add_preconnect_to_head( $items );
 	}
+
+	/**
+	 * Stop CDN from adding resource hints into head.
+	 *
+	 * @param bool $status Current status.
+	 * @return bool
+	 */
+	public function stop_cdn_insert_resource_hints( $status ) {
```

Comment:
Let's try to keep up with code standards :sweat_smile: 
```suggestion
	public function stop_cdn_insert_resource_hints( bool $status ): bool {
```